site/docs/pages/identity/address.mdx

```diff
@@ -16,21 +16,32 @@ import { Address } from '@coinbase/onchainkit/identity';
 ```
 
 <App>
-  <Address  address="0x02feeb0AdE57b6adEEdE5A4EEea6Cf8c21BeB6B1" />
+  <Address className="bg-gray-300 rounded" address="0x02feeb0AdE57b6adEEdE5A4EEea6Cf8c21BeB6B1" />
+</App>
+
+Set `isSliced` to false, to display the full address:
```

Comment:
One more polish way to write docs that I learnt recenlty is the usage of `###` to create sub section with short titles.

Like we did for the https://onchainkit.xyz/swap/swap docs.

so we have something like
- [Custom token pair](https://onchainkit.xyz/swap/swap#custom-token-pair)
- [Remove toggle button](https://onchainkit.xyz/swap/swap#remove-toggle-button)
- [Remove swap message](https://onchainkit.xyz/swap/swap#remove-swap-message)
- [Override styles](https://onchainkit.xyz/swap/swap#override-styles)

Try to do the same with those pages.

Reply:
That's very nice. Updating